spec/displaynames.html

```diff
@@ -31,6 +38,82 @@ <h1>CanonicalCodeForDisplayNames ( _type_, _code_ )</h1>
     </emu-clause>
   </emu-clause>
 
+    <emu-clause id="sec-isvaliddatetimefieldcode" type="abstract operation">
+      <h1>
+        IsValidDateTimeFieldCode (
+          _field_: a String,
+        )
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It verifies that the _field_ argument represents a valid date time field code.</dd>
+      </dl>
+      <emu-alg>
+        1. If _field_ is listed in the Code column of <emu-xref href="#table-validcodefordatetimefield"></emu-xref>, return *true*.
+        1. Return *false*.
+      </emu-alg>
+
+      <emu-table id="table-validcodefordatetimefield">
+        <emu-caption>Codes For Date Time Field of DisplayNames</emu-caption>
+        <table class="real-table">
+          <thead>
+            <tr>
+              <th>Code</th>
+              <th>Description</th>
+            </tr>
+          </thead>
+          <tr>
+            <td>*"era"*</td>
+            <td>The field indicating the era, e.g. AD or BC in the Gregorian (Julian) calendar.</td>
+          </tr>
+          <tr>
+            <td>*"year"*</td>
+            <td>The field indicating the year.</td>
```

Comment:
Should this be more specific?
```suggestion
            <td>The field indicating the year (within an era).</td>
```